src/Elders.Cronus/Projections/ProjectionVersions.cs

```diff
@@ -41,55 +41,52 @@ public void ValidateVersion(ProjectionVersion version)
             }
         }
 
-        public ProjectionVersions WithoutTheGarbage()
+        public void Add(ProjectionVersion version)
```

Comment:
Not sure If I like this being 'Add' while still having some side-effects. Maybe a better name?

Reply:
Suggestions?

Reply:
In general, we should revisit this at some later stage where i have more knowledge of how this should work and then separate the add from the sanitizing. The best solution for now should be 'Upsert' with a summary about what the method actually does.